### DISCUSSION TITLE
Need help setting DecapCMS with private github Repos

### DISCUSSION THREAD

**Discussion by RProduction** [decaporg/decap-cms#6961] [Q&A]
Hello, I need help with setting DecapCMS with private Github repository.
I had successfully ingrated the CMS with my site, but there is a problem.
For some reason, when I save the content in CMS, the CMS create a branch and I need to manually pull request the branch.
Is there is a way to stop this? I am using DecapCMS with editorial workflow with Sveltekit site.

For the error this is the error:
`
{
    "type": "UNPUBLISHED_ENTRY_PERSIST_FAILURE",
    "payload": {
        "error": {
            "message": "Validation Failed",
            "status": 422,
            "api": "GitHub",
            "meta": {},
            "name": "API_ERROR"
        },
        "collection": "novel",
        "slug": ""
    },
    "error": {
        "message": "Validation Failed",
        "status": 422,
        "api": "GitHub",
        "meta": {},
        "name": "API_ERROR"
    }
}
`

**Comment by naturedamends** [comment]
set `editorial_workflow: false` in your yaml. 
The editorial workflow creates pull requests for each file edit.

I've also used `show_preview_links: false`

With editorial workflow, you can merge the pull request by publishing the post. It needs to be in the "ready" state. So not sure if thats your error. Not affiliated with decap

**Reply by RProduction** [reply]
I am sorry for this but probably I test the CMS in localhost.
